docs/source/install.rst

```diff
@@ -109,11 +109,11 @@ These optional dependencies and their minimum supported versions are listed belo
 +------------------+-----------------+---------------------------------------------------------------------------------------------------------+
 | `crick`_         | ``>=0.0.3``     | Use ``tdigest`` internal method for dataframe statistics computation                                    |
 +------------------+-----------------+---------------------------------------------------------------------------------------------------------+
-| `cytoolz`_       | ``>=0.10.1``    | Faster cythonized implementation of internal iterators, functions, and dictionaries                     |
+| `cytoolz`_       | ``>=0.11.0``    | Faster cythonized implementation of internal iterators, functions, and dictionaries                     |
```

Comment:
Changes to this table are causing the docs build to fail (xref https://readthedocs.org/projects/dask/builds/20736441/) 

```
/home/docs/checkouts/readthedocs.org/user_builds/dask/checkouts/10295/docs/source/install.rst:101: ERROR: Malformed table.
```